OpenIGTLinkIF/MRML/vtkMRMLIGTLConnectorNode.h

```diff
@@ -296,6 +298,11 @@ class VTK_SLICER_OPENIGTLINKIF_MODULE_MRML_EXPORT vtkMRMLIGTLConnectorNode : pub
   //ETX
 #endif // __VTK_WRAP__
 
+  // Outgoing header version
+  // Should be set before messages are sent
+  vtkGetMacro(HeaderVersion, int);
```

Comment:
It should be called something like OutgoingMessageHeaderVersionMaximum to make it clear what it is used for. By default it could be set to (-1), which would mean "no limit" and that value would not be written to the scene (only positive values).